src/main/java/com/pollistics/services/PollService.java

```diff
@@ -23,7 +23,12 @@ public Poll getPoll(String id) {
 
 	public String createPoll(String name, HashMap<String, Integer> options) {
 		Poll poll = pollRepo.insert(new Poll(name, options));
-		return poll.getId();
+		return poll.getSlug();
```

Comment:
Not sure, the slug is more legible I guess so I picked this one to do by default